tests/storage/migration.go

```diff
@@ -70,7 +70,7 @@ import (
 	"kubevirt.io/kubevirt/tests/testsuite"
 )
 
-var _ = SIGDescribe("Volumes update with migration", decorators.RequiresTwoSchedulableNodes, decorators.VMLiveUpdateRolloutStrategy, Serial, func() {
+var _ = SIGDescribe("Volumes update with migration", decorators.RequiresTwoSchedulableNodes, decorators.VMLiveUpdateRolloutStrategy, func() {
```

Comment:
I feel like I'm missing something, the BeforeEach still edits to kubevirt CR in this revision, so it can't run in parallel to others, no?

Reply:
I took a look at the cpu and memory hotplug and they are run in parallel, even if they patch kubevirt CR. You can see that we have a label for the rollout strategy for those tests.

Reply:
/hold

>I took a look at the cpu and memory hotplug and they are run in parallel, even if they patch kubevirt CR. You can see that we have a label for the rollout strategy for those tests.

That is wrong as well

Reply:
Ah, instancetype also does uses the same mechanism. TBH, I think we could simply enable it by default and do the patch for the stage strategy,

Reply:
@xpivarc does the hold still stand?

Reply:
VM live update has become the default, so it should be fine now

Reply:
I guess we can undraft/hold now ping @xpivarc